aea/helpers/base.py

```diff
@@ -295,7 +295,6 @@ def wrapper(*args, **kwargs):
             try:
                 return fn(*args, **kwargs)
             except Exception as e:  # pylint: disable=broad-except  # pragma: no cover  # generic code
-                logger.exception(e)
```

Comment:
it was added for debugging purposes; no need to have it (although it could still be useful to have the exception trace in the output)

Reply:
Yes very anoying; it needs to be removed otherwise it will confuse the user.